src/browserlib/extract-dfns.mjs

```diff
@@ -68,6 +68,7 @@ function hasValidType(el) {
     'setlike',
     'extended-attribute',
     'event',
+    'permission',
```

Comment:
Could you also update the comment on line 37 to add "permission" to the list of types not yet defined in Bikeshed's doc?

Reply:
done in 7889fb8